examples/reference-entity-model/src/main/ml-config/security/users/hub-developer.json

```diff
@@ -4,6 +4,7 @@
   "password": "%%testPassword%%",
   "role": [
     "data-hub-developer",
-    "hub-central-user"
+    "hub-central-user",
```

Comment:
I think data-hub-developer should gain the data-hub:writeMapping privilege, which then means we don't have to modify these users or the cypress users that already have data-hub-developer. That privilege can then be used in xdmp.securityAssert calls in the DS endpoints.